src/realm/group.hpp

```diff
@@ -1002,7 +1002,7 @@ inline ref_type Group::get_history_ref(const Array& top) noexcept
 inline int Group::get_history_schema_version(const Array& top) noexcept
 {
     if (top.is_attached()) {
-        if (top.size() >= 8) {
+        if (top.size() >= 9) {
```

Comment:
Here is the new commit, https://github.com/realm/realm-core/pull/2549/commits/a1d1f0462cec4b8b5e45a5ebaa646570f64efdb7.